lib/beaker/dsl/helpers/web_helpers.rb

```diff
@@ -14,10 +14,11 @@ def port_open_within?( host, port = 8140, seconds = 120 )
 
         #Determine is a given URL is accessible
         #@param [String] link The URL to examine
+        #@param [Integer] limit redirect limit
         #@return [Boolean] true if the URL has a '200' HTTP response code, false otherwise
```

Comment:
```suggestion
        #@return [Boolean] true if the ultimate URL after following redirects (301&302) has a '200' HTTP response code, false otherwise
```